docs/src/tutorial.md

```diff
@@ -302,3 +302,15 @@ parameters = Dict(:min_to_be_happy=>collect(2:5), :numagents=>[200,300], :griddi
 data = paramscan(parameters, initialize;
        properties=properties, n = 3, agent_step! = agent_step!)
 ```
+
+We can combine all replicates with an aggregating function, such as mean, using
+the `aggregate` function from the `DataFrames` package:
+
+```@example schelling
+import Statistics: mean
+data_mean = Agents.aggregate(data, [:step, :min_to_be_happy, :numagents],  mean)
```

Comment:
How much sense does it make to average over `:step` ? I mean scientifically, not programmatically?